api-spec/api.yaml

```diff
@@ -434,6 +473,51 @@ components:
         - ENABLED
         - DISABLED
         - INCOMING
+    PreauthorizationResponse:
+      type: object
+      description: Form data needed to create a payment method input form
+      properties:
+        sessionId:
+          type: string
+          description: Identifier of the payment gateway session associated to the form
+        fields:
+          oneOf:
+            - $ref: "#/components/schemas/CardFormFields"
+          discriminator:
+            propertyName: paymentMethod
+            mapping:
+              CARD: "#/components/schemas/CardFormFields"
+      required:
+        - fields
+        - sessionId
+    CardFormFields:
+      type: object
+      description: Form fields for credit cards
+      properties:
+        paymentMethod:
+          type: string
+        form:
+          type: array
+          items:
+            $ref: '#/components/schemas/Field'
+      required:
+        - paymentMethod
+        - form
+    Field:
+      type: object
+      properties:
+        type:
+          type: string
```

Comment:
What do you think about add format: uri specification on those fields?

Reply:
Fixed in ee6c47a